src/middleware/validations.ts

```diff
@@ -196,12 +197,9 @@ export default class Validations {
 			body('childInterest').isString(),
 			body('wishItemName').notEmpty().withMessage('Wish item name is required').isString(),
 			body('wishItemPrice').notEmpty().withMessage('Wish item price is required').isNumeric(),
-			body('wishItemURL')
-				.notEmpty()
-				.withMessage('Wish item url is required')
-				.isString()
-				// https://regex101.com/r/yM5lU0/13 if you want to see it in action
-				.matches(/^(https?(:\/\/)){1}([w]{3})(\.amazon\.com){1}\/.*$/)
+			amazonValidator
+				.body('wishItemURL')
+				.isValidLink()
 				.withMessage('Wish item url has to be a valid amazon link!'),
```

Comment:
Could you change this to "Item URL must start with https://www.amazon.com/ to be a valid Amazon link."

I think more information helps the user to fix their error.

Reply:
Done :)